src/tokenizer/ngram_tokenizer.rs

```diff
@@ -0,0 +1,121 @@
+use super::{Token, TokenStream, Tokenizer};
+
+///Tokenize the text by splitting words into ngrams of the given size
+#[derive(Clone)]
+pub struct NgramTokenizer {
+  min_gram: usize,
+  max_gram: usize,
+  edges_only: bool,
```

Comment:
Should `edges_only` be called `prefix_only`?
I don't think the code does suffix right now, does it?

Reply:
It does not.  I was following the naming of [Lucene](https://github.com/apache/lucene-solr/blob/master/lucene/analysis/common/src/java/org/apache/lucene/analysis/ngram/NGramTokenizer.java#L69) - I'm happy to name it what makes sense for tantivy though.